tests/test_data/test_pipelines/test_transform/test_transform.py

```diff
@@ -79,6 +79,28 @@ def test_resize():
     assert np.equal(results['img'], results['img2']).all()
     assert results['img_shape'] == (800, 1280, 3)
 
+    results_seg = {
+        'img': img,
+        'img_shape': img.shape,
+        'ori_shape': img.shape,
+        'seg1': copy.deepcopy(img),
+        'seg2': copy.deepcopy(img),
```

Comment:
We should use better names of keys.
Use `gt_semantic_seg` and `gt_seg` instead of `seg1` and `seg2` respectively will be better.